### PR TITLE
Add Speedtest-cli optional component

### PR DESCRIPTION
**Pull request by kubawolanin**
Signed-off-by: Kuba Wolanin <hi@kubawolanin.com>

### CODE REVIEW

**Comment by ThomDietrich** [comment]
Hey Kuba, cool addition! I wonder why I did never add it myself.
Fun fact: The speedtest tutorial was the first tutorial I wrote in the community.

File: functions/packages.sh [hunk 1]
```diff
@@ -381,6 +381,25 @@ The article also contains instructions regarding openHAB integration.
...
+  failtext="Sadly there was a problem setting up the selected option. Please report this problem in the openHAB community forum or as a openHABian GitHub issue."
+  successtext="Setup successful. Please continue with the instructions you can find here:\n\nhttps://community.openhab.org/t/7611/1"
+
+  echo "$(timestamp) [openHABian] Setting up Speedtest CLI... "
+
+  cond_redirect apt -y install python-setuptools || FAILED=1
+  cond_echo ""
+  echo "Speedtest CLI... "
+  cond_redirect easy_install speedtest-cli || FAILED=1
+  cond_redirect apt update || FAILED=1
```
Comment (ThomDietrich) [inline functions/packages.sh]: Why did you add this line?

Comment (kubawolanin) [reply functions/packages.sh]: Thought it's a common step in these components - I saw it e.g. [here](https://github.com/openhab/openhabian/pull/421/files#diff-c95dc1071504d576bc897b9f25cca552L406) 🙈

Comment (ThomDietrich) [reply functions/packages.sh]: Ah... but then it seems like you shuffled the order by accident!?

```bash
cond_redirect apt update || FAILED=1
cond_redirect apt -y install python-setuptools || FAILED=1
cond_echo ""
echo "Speedtest CLI... "
cond_redirect easy_install speedtest-cli || FAILED=1
```

**Comment by ThomDietrich** [comment]
Hey @kubawolanin,
I've updated your PR. Please have a look, thanks mate ;)

Btw. YES, the use of FAILED is inconsistent. I started to introduce it at a certain point but did not yet consistently follow through.
Atm I do not have the time to invest serious work here. At a later point I'd like to discuss a new strategy and future plans for openHABian. I was e.g. thinking about replacing (at least) optional components routines by Ansible. Jm2c

**Comment by ThomDietrich** [comment]
Hey Kuba,

I have no experience with Ansible (or one of it's brothers and sisters) either. However it's just the logical step we should have gone a while ago. Bash is not meant for super flexible routine work. I've already started to solve things be introducing stuff like `cond_redirect` or `FAILURE` but bash can only bring us this far. Ansible is meant for this kind of things. See: https://github.com/rvben/ansible-speedtest/blob/master/speedtest.yml

Yes I'll merge now.

Regarding openHAB integration: Many of the optional components and also other menu options would benefit from a way to interact with openHAB. I'd like to come up with a set o functions to e.g. check if openHAB is online, install bindings, inject files into `/etc/openhab2` (with proper testing), and more. 

For a first step see: https://github.com/openhab/openhabian/issues/419
